src/pages/FormPage.tsx

```diff
@@ -1,170 +1,275 @@
-    setSubmitted(true)
+    setSubmitted(true);
     setTimeout(() => {
-        document.querySelector('tds-toast')?.hideToast()
-    }, 5000)
+      document.querySelector("tds-toast")?.hideToast();
+    }, 5000);
   };
 
   return (
     <>
-    <article>
-      <form
-        ref={form}
-        onSubmit={(event: FormEvent) => {
-          handleSubmit(event);
-        }}
-      >
-        <h3>Form example</h3>
-        <section>
-          <h4>Text Input</h4>
-          <tds-text-field
-            name="text-field"
-            label="Full name"
+      <article>
+        <form
+          ref={form}
+          onSubmit={(event: FormEvent) => {
+            handleSubmit(event);
+          }}
+        >
+          <h3>Form example</h3>
+          <section>
+            <h4>Text Input</h4>
+            <tds-text-field
+              name="text-field"
+              label="Full name"
+              label-position="outside"
+              placeholder="John Doe"
+            ></tds-text-field>
+            <tds-datetime
+              label="Date of birth"
+              name="dateOfBirth"
+              type="date"
+              default-value="2000-01-01"
+            ></tds-datetime>
+          </section>
+
+          <section>
+            <tds-dropdown
+              ref={countryDropdown}
+              id="country-dropdown"
+              size="lg"
+              placeholder="Please select country of your choice"
+              open-direction="auto"
+              label-position="outside"
+              label="Current country"
+              type="default"
+            >
+              <tds-dropdown-option value="sweden" tabIndex={0}>
+                Sweden
+              </tds-dropdown-option>
+              <tds-dropdown-option value="norway" tabIndex={0}>
+                Norway
+              </tds-dropdown-option>
+              <tds-dropdown-option value="finland" tabIndex={0}>
+                Finland
+              </tds-dropdown-option>
+            </tds-dropdown>
+          </section>
+
+          <section>
+            <tds-dropdown-filter
+              ref={norwayDropdownTown}
+              size="lg"
+              placeholder="Norweigan Town"
+              disabled={countrySelected !== "norway"}
+              open-direction="auto"
+              label-position="outside"
+              label="In case you are from Norway, which town?"
+              data={JSON.stringify(townDataNorway)}
+            ></tds-dropdown-filter>
+          </section>
+
+          <section>
+            <tds-dropdown-v2
+              ref={swedenDropdownTown}
+              name="dropdown"
+              label="In case you are from Sweden, which town?"
+              label-position="outside"
+              placeholder="Swedish Town"
+              helper=""
+              size="lg"
+              open-direction="auto"
+              filter
+              disabled={countrySelected !== "sweden"}
+            ></tds-dropdown-v2>
+          </section>
```

Comment:
```suggestion
          <section>
            <tds-dropdown-v2
              ref={norwayDropdownTown}
              name="town"
              label="In case you are from Norway, which town?"
              label-position="outside"
              placeholder="Norweigan Town"
              helper=""
              size="lg"
              open-direction="auto"
              filter
              disabled={countrySelected !== "norway"}
            ></tds-dropdown-v2>
          </section>

          <section>
            <tds-dropdown-v2
              ref={swedenDropdownTown}
              name="town"
              label="In case you are from Sweden, which town?"
              label-position="outside"
              placeholder="Swedish Town"
              helper=""
              size="lg"
              open-direction="auto"
              filter
              disabled={countrySelected !== "sweden"}
            ></tds-dropdown-v2>
          </section>
```

Reply:
I think one of them should use the <tds-dropdown-option-v2> elements